lib/vector-buffers/src/config.rs

```diff
@@ -293,6 +292,11 @@ impl BufferType {
 /// component, where you could only choose which buffer type to use.  As we expand buffer
 /// functionality to allow chaining buffers together, you'll see "buffer topology" used in internal
 /// documentation to correctly reflect the internal structure.
+///
+/// TODO: We need to limit chained buffers to only allowing a single copy of each buffer type to be
```

Comment:
Is this still relevant?

Reply:
Yeah, it's relevant.  I'll make an actual issue for it so that this TODO isn't lost to the sands of git history.

Reply:
Opened #12123 for this.